app/views/content_items/manuals/_updates.html.erb

```diff
@@ -18,30 +18,54 @@
             font_size: "l"
           } %>
 
-          <%= render "govuk_publishing_components/components/accordion", {
-            heading_level: 3,
-            items: updates_by_year.map do |day, updated_documents|
-              accordion_content = capture do %>
-                <div class="govuk-!-margin-top-3">
-                  <% updated_documents.each do |_, change_notes| %>
-                    <% change_notes.each do |change_note| %>
-                      <p class="govuk-body"><%= link_to change_note["title"], change_note["base_path"], class: "govuk-link" %></p>
+          <div data-module="gem-track-click gtm-click-tracking">
```

Comment:
The module we're adding is `gtm-click-tracking`, so I'm not sure why `gem-track-click` has also been added here?

Also it is also possible to pass `gtm-click-tracking` as an option to the component directly, instead of wrapping the whole block in a new DIV. Example:

```
<%= render "govuk_publishing_components/components/accordion", {
  heading_level: 3,
  data_attributes: {
    module: "gtm-track-click",
    ga4: {
    (etc.)
```

Reply:
Thanks @andysellick  - quick one - as we are adding this to all `data_attributes` objects for all accordions, does the `module` attribute also need to be added to the show-all `data_attributes` object too? If so, does it look like this...

```erb
<%= render "govuk_publishing_components/components/accordion", {
  heading_level: 3,
  data_attributes_show_all: {
    "module": "gtm-track-click",
    "ga4": show_all_attributes.to_json
  },
} %>

```

Reply:
No, it's only an option to pass to `data_attributes`. Once there it'll be added to the component as below, which will mean that the `gtm-track-click` script gets activated for anything clicked within the component.

```
<div class="gem-c-accordion" data-module="gtm-track-click">
   <!-- contents of accordion -->
</div>
```

Reply:
Great - thanks @andysellick 👍